server/src/main/java/org/elasticsearch/action/search/QueryPhaseResultConsumer.java

```diff
@@ -106,7 +105,7 @@ public QueryPhaseResultConsumer(
 
     @Override
     public void close() {
-        Releasables.close(pendingMerges);
+        pendingMerges.close();
```

Comment:
Just wondering why the try/catch of the IOException in Releasables.close() isn't needed anymore. Is it because QueryPhaseConsumers are already closed via Releasables#close in AbstractSearchAsyncAction#releaseContext or because of some refactoring in the lambdas elsewhere in this PR?

Reply:
`Releasables.close`'s IO exception catching is actually dead code :P a `Releasable` never throws an IOException to begin with, the code is just there because of laziness so that we can reuse `IOUtils` which does the same thing for `Closeable`. I just ditched the Releasables.close here to make the fact that `pendingMerges` is never null visible :)